package.json

```diff
@@ -31,6 +31,7 @@
     "ember-ajax": "2.0.1",
     "ember-autosave": "2.0.1",
     "ember-bootstrap": "0.7.2",
+    "ember-browserify": "^1.1.13",
```

Comment:
We've discussed this [before](https://github.com/CenterForOpenScience/ember-preprints/pull/269#issuecomment-28444624), it seems like ember-browserify is needed to use some Ember-OSF dependencies. It doesn't matter that ember-browserify is installed in ember-osf; you need it in the consuming application.  I think it's a bug with [ember-browserify and addons](https://github.com/ef4/ember-browserify#using-ember-browserify-in-addons)